src/libsyncengine/requests/serverrequests.cpp

```diff
@@ -985,6 +986,120 @@ ExitCode ServerRequests::getUserFromSyncDbId(int syncDbId, User &user) {
     return ExitCodeOk;
 }
 
+ExitCode ServerRequests::sendLogToSupport(bool includeArchivedLog, std::function<bool(char, int)> progressCallback,
+                                          ExitCause &exitCause) {
+    exitCause = ExitCauseUnknown;
+    ExitCode exitCode = ExitCodeOk;
+    std::function<bool(char, int)> safeProgressCallback =
+        progressCallback != nullptr ? std::function<bool(char, int)>([progressCallback](char status, int percent) { return progressCallback(status, percent); })
+                                    : std::function<bool(char, int)>([](char, int) { return true; });
+    
+    safeProgressCallback('A', 0);
```

Comment:
Declare an enum for the status values

Reply:
I've done this implementation in commit: [0d4fdd6](https://github.com/Infomaniak/desktop-kDrive/pull/117/commits/0d4fdd6adf8411e6349bd43cc8113c68f131992d)